doc/content/getting-started/migrating/migrating-from-v2/migrate-using-migration-tool.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Migrate using Migration Tool"
+description: ""
+weight: 2
+---
+
+This section refers to migrating end devices to {{% tts %}} using the `ttn-lw-migrate` tool.
+
+<!--more-->
+
+Follow the steps below to migrate a single end device or a batch of devices from {{% ttnv2 %}} to {{% tts %}} using the `ttn-lw-migrate` migration tool.
+
+`ttn-lw-migrate` tool is used to export end devices from {{% ttnv2 %}} to a [JSON file]({{< ref "/getting-started/migrating/device-json" >}}). This JSON file can afterwards be imported in {{% tts %}} via Console or via CLI.
+
+## Prerequisites
+
+1. [`ttn-lw-migrate` tool](https://github.com/TheThingsNetwork/lorawan-stack-migrate) installed on your system.
```

Comment:
Please add a note that version ttn-lw-migrate v0.5.0 or newer is required, as well as The Things Stack version 3.12. Probably something along the lines of "The commands will not work with older versions of ttn-lw-migrate or The Things Stack, and importing the devices will fail."

Reply:
Isn't session migration what's available from 3.12? I mean, will importing device without a session work for a version of TTS lower than v3.12? The same question for ttn-lw-migrate v0.5.0?

Reply:
This is not only about session vs no-session. The API changes in 3.12 and previous versions of ttn-lw-migrate no longer work.